modules/dropdown/test/spec.coffee

```diff
@@ -61,14 +58,13 @@ describe 'hx-dropdown', ->
     hx.loop = hx_loop = (f) ->
       g = -> hx_loop_update(f, g)
       hx_loop_update(f, g)
-    jasmine.clock().install()
     baseTime = new Date(2013, 0, 1)
-    jasmine.clock().mockDate(baseTime)
+    clock = sinon.useFakeTimers(baseTime.getTime())
```

Comment:
Is setting the baseTime actually required here? It shouldn't matter when the clock runs and `useFakeTimers` seems to default to 0 rather than 'now' so already has a consistent start point